Oluwasegun-AA/Swift-Foods
# 162163335 Setup react redux

#### What does this PR do?
This PR allows a user to get likes details for articles

#### Description of Task to be completed?
* Setup React redux

#### How should this be manually tested?
* clone the repository  ` https://github.com/shegsteham/swift-foods.git `
* cd to the  ` swift-food ` folder
* check out the`Ch-setup-react-redux-162163335` branch
* run `npm i` on the bash terminal
* run npm run start

#### What are the relevant pivotal tracker stories?
#162163335

#### Any background context you want to add?
N/A

#### Screenshots
